src/installer.ts

```diff
@@ -47,7 +47,7 @@ export async function getMage(version: string): Promise<string> {
 
 const getFilename = (semver: string): string => {
   const platform: string = osPlat == 'win32' ? 'Windows' : osPlat == 'darwin' ? 'macOS' : 'Linux';
-  const arch: string = osArch == 'x64' ? '64bit' : '32bit';
+  const arch: string = osArch == 'x64' || osArch == 'arm64' ? '64bit' : '32bit';
```

Comment:
Looking at [latest release](https://github.com/magefile/mage/releases/tag/v1.14.0), I think it should be:

```suggestion
  const arch: string = osArch == 'x64' || osArch == 'ARM64' ? '64bit' : '32bit';
```

Reply:
Ah, yea, thanks. Done.

Reply:
I guess there might need to be a handling of `arm32 -> ARM` as well, but I'm assuming that 32-bit ARM is what the `-ARM` suffix is about, and I also don't know that `uname -m` is `arm32` on those systems. Something to look into, or you happy with this as is?